app/reports_generation/scripts/chart_generator.py

```diff
@@ -22,10 +30,46 @@ def __resolve_and_expand_user_path(path: Path) -> Path:
     return path.resolve().expanduser()
 
 
+def __get_all_default_actions():
```

Comment:
This function is executed in a loop for every line, but read actions list should be a one-time operation.